skorch/utils.py

```diff
@@ -128,10 +128,26 @@ def to_numpy(X):
 def to_device(X, device):
     """Generic function to move module output(s) to a device.
 
-    Deals with X being a torch tensor or a tuple of torch tensors.
+    Parameters
+    ----------
+    X : input data
+        Deals with X being a:
+        * torch tensor
+        * tuple of torch tensors
+        * PackSequence instance
+        * torch.nn.Module
 
+    device : str, torch.device
+        The compute device to be used. If device="auto" we set to
```

Comment:
Although we're not 100% consistent with this, we generally don't use "we" in the docstrings

```suggestion
        The compute device to be used. If device="auto", it is set to
```

Reply:
Ah okay, thanks for letting me know. I've gone with your recommendation.